its/src/test/java/com/sonar/it/scanner/msbuild/ScannerMSBuildTest.java

```diff
@@ -595,13 +595,54 @@ public void testEsprojVueWithBackend() throws IOException {
 
     List<Issue> issues = TestUtils.allIssues(ORCHESTRATOR);
     List<String> ruleKeys = issues.stream().map(Issue::getRule).collect(Collectors.toList());
-    assertThat(ruleKeys).containsAll(Arrays.asList("csharpsquid:S4487", "csharpsquid:S1134"));
+    assertThat(ruleKeys).containsAll(Arrays.asList(SONAR_RULES_PREFIX + "S4487", SONAR_RULES_PREFIX + "S1134"));
 
     assertThat(TestUtils.getMeasureAsInteger(localProjectKey, "lines", ORCHESTRATOR)).isEqualTo(74);
     assertThat(TestUtils.getMeasureAsInteger(localProjectKey, "ncloc", ORCHESTRATOR)).isEqualTo(53);
     assertThat(TestUtils.getMeasureAsInteger(localProjectKey, "files", ORCHESTRATOR)).isEqualTo(3);
   }
 
+  @Test
+  public void testCSharpSdk7() throws IOException {
+    ORCHESTRATOR.getServer().provisionProject(PROJECT_KEY + ".15", "CSharp.SDK.7.0");
+
+    if (!TestUtils.getMsBuildPath(ORCHESTRATOR).toString().contains("2022")) {
+      return; // This test is not supported on versions older than Visual Studio 22
+    }
+
+    BuildResult result = runBeginBuildAndEndForStandardProject("CSharp.SDK.7.0", "");
+    assertTrue(result.isSuccess());
+
+    List<Issue> issues = TestUtils.allIssues(ORCHESTRATOR);
+
+    if (isTestProjectSupported()) {
```

Comment:
I opened https://github.com/SonarSource/sonar-scanner-msbuild/issues/1348 to remove this